auto_submit/lib/requests/github_webhook.dart

```diff
@@ -59,7 +60,7 @@ class GithubWebhook extends RequestHandler {
 
     if (hasAutosubmit || hasRevertLabel) {
       log.info('Found pull request with auto submit and/or revert label.');
-      await pubsub.publish('auto-submit-queue', pullRequest);
+      await pubsub.publish(Config.pubsubPullRequestTopic, pullRequest);
```

Comment:
is this a static property? or should we use an instance of config rather than the class?

Reply:
It is currently static. We have a mix but I can make it an instance field if that is preferable?